src/ci/docker/test-various/Dockerfile

```diff
@@ -20,10 +23,18 @@ RUN curl -sL https://github.com/denzp/rust-ptx-linker/releases/download/v0.9.0-a
 RUN curl -sL https://nodejs.org/dist/v9.2.0/node-v9.2.0-linux-x64.tar.xz | \
   tar -xJ
 
+WORKDIR /build/
+COPY scripts/musl-toolchain.sh /build/
+RUN CFLAGS="-Wa,-mrelax-relocations=no" \
+    CXXFLAGS="-Wa,-mrelax-relocations=no" \
```

Comment:
I think technically it's fine to omit this for this container since it's just running tests, but it shouldn't matter either way.